cmd/cluster/cluster.go

```diff
@@ -18,6 +18,7 @@ package cluster
 import (
 	"github.com/spf13/cobra"
 	"github.com/yugabyte/ybm-cli/cmd/cluster/cert"
+	encryption "github.com/yugabyte/ybm-cli/cmd/cluster/encryption"
```

Comment:
Why do you alias it to "encryption" and sometime "ear" alias should be use mostly if there is a risk of confusion or already existing package with the same name

Reply:
Sorry, I will make it consistent. I initially called it "EAR", but PM feedback was that "EAR" does not mean much and should rename to encryption.